<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowEmployee</class>
 <widget class="QMainWindow" name="WindowEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о сотруднике</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Водитель</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фамилия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отчество:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер телефона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 }

QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; 
     padding:2 13px;
 }  </string>
    </property>
    <property name="title">
     <string>В данный момент находится в рейсе:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Номер маршрута:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Автобус:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumberRoute">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumberBus">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLastName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Измайлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFirstName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Семен</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTelephone">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8-923-923-00-00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddress">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ул. Некрасова 141 кв. 7537</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPatronymic">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Егорович</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionCall"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Создать / Изменить путевой лист</string>
   </property>
  </action>
  <action name="actionCall">
   <property name="text">
    <string>Связаться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
